packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -118,6 +118,25 @@ const installDependencies = async (
   }
 };
 
+// Function to update package.json engines
+const updatePackageEngines = (targetDirectory: string) => {
+  const packageJsonPath = path.resolve(targetDirectory, "package.json");
+
+  try {
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf-8"));
+    let engines = packageJson.engines;
+    if (!engines) {
+      engines = { node: "" };
+    }
+    engines.node = "^18.0.0 || >=20.0.0";
```

Comment:
Kind of strange that we support 18 and 20+, but not 19

Reply:
We are support what Vite supports. Odd versions of Node are quick betas. Once the next even version is released the prior odd becomes deprecated, with the previous even being in LTS.

Reply:
https://nodejs.org/en/about/previous-releases